nautobot_golden_config/models.py

```diff
@@ -486,21 +495,33 @@ def get_absolute_url(self):  # pylint: disable=no-self-use
 
     def __str__(self):
         """Return a simple string if model is called."""
-        return "Golden Config Settings"
+        return "Configuration Object"
```

Comment:
The UI representation was weird, this was my suggestion. It would render in the edit page it was `Golden Config Settings Golden Config Settings`. Which is <obj.Meta.verbose_plural_name> <instance.__str__>.